setup.py

```diff
@@ -16,17 +16,19 @@
 from setuptools import find_packages
 from setuptools import setup
 
+exec(open('opencensus/common/version/__init__.py').read())
+
 extras = {
-    "stackdriver": ['google-cloud-trace>=0.20.1, <0.30'],
-    "prometheus_client": ['prometheus_client==0.3.1'],
-    "requests": ['wrapt==1.10.11']
+    'prometheus_client': ['prometheus_client==0.3.1'],
+    'requests': ['wrapt==1.10.11'],
+    'stackdriver': ['google-cloud-trace>=0.20.1, <0.30'],
 }
 
 install_requires = [
+    'opencensus-correlation == 0.2.dev0',
```

Comment:
when we try to install opencensus at this commit or newer, we get an error:
```
Obtaining opencensus from git+https://github.com/census-instrumentation/opencensus-python.git@d37b6d267307a136631881e593c2bc8921a786b6#egg=opencensus
  Skipping because already up-to-date.
Collecting opencensus-correlation==0.2.dev0 (from opencensus)

  Could not find a version that satisfies the requirement opencensus-correlation==0.2.dev0 (from opencensus) (from versions: )
No matching distribution found for opencensus-correlation==0.2.dev0 (from opencensus)
```

Has any version of it been released yet? Should it be required or an extra? I'm not totally clear on the new packaging story, and know that our use case of installing for git isn't necessary supported, but want to know if we can make it work until there's a new release with #534 included :grinning:

Reply:
@colincadams Sorry for the inconvenience! It hasn't been released yet. OpenCensus will have dependency on `opencensus-correlation` and `opencensus-tracecontext` (not there yet), the other contrib packages (e.g. extensions, exporters) will have dependency on OpenCensus.

The workaround is to install the package from a cloned git repo, please refer to
https://github.com/census-instrumentation/opencensus-python/blob/master/nox.py#L21